content/about/contact.mdx

```diff
@@ -9,27 +9,38 @@ description: We provide support for researchers seeking help with statistical mo
 The Center for Computation and Visualization (CCV) is a center within the University's central IT organization, which is the Office of Information Technology (OIT). In addition to building and maintaining the University's hundreds of enterprise software, systems, and hardware, OIT is also responsible for driving the technological progress that enables scientific research. Executing on the University's research mission is the key role that CCV plays in OIT.
 
     <CardGroup>
-        <StyledCard title="Email" iconName="FaEnvelopeOpenText">
+        <StyledCard
+            title="Email"
+            iconName="FaEnvelopeOpenText"
+            footer={<Button href="mailto:support@ccv.brown.edu">Email Us</Button>}>
+
             Email us at <CopyableEmail email="support@ccv.brown.edu"/> to inquire about resources, support, or a potential collaboration for a research project.
+        </StyledCard>
+
+        <StyledCard
+            title="Submit a Ticket"
+            iconName="FaTicketAlt"
+            footer={<Button href="mailto:support@ccv.brown.edu">Submit a Ticket</Button>}>
 
-            <Button href="mailto:support@ccv.brown.edu">
-                Email Us
-            </Button>
+            Open a ticket with our user services team. We will respond as soon as possible.
         </StyledCard>
 
-        <StyledCard title="CCV Slack" iconName="FaSlack">
+        <StyledCard
+            title="CCV Slack"
+            iconName="FaSlack"
+            footer={
```

Comment:
<img width="1815" height="1055" alt="Screenshot 2025-09-16 at 2 26 08 PM" src="https://github.com/user-attachments/assets/0e0abf0d-e458-4e84-9834-3741e2a3de5d" />
should teh buttons be centered?